src/components/Image/Image.jsx

```diff
@@ -14,6 +15,7 @@ function ImageAmp ( {
 	return (
 		<amp-img
 			alt={alt}
+			className={className}
```

Comment:
so this was a bug and wasn't being passed before?

Reply:
perhaps not actually, looking at Image.jsx in qz-react I see we were never passing a className to amp-img. I'll remove this